HACKING.rst.txt

```diff
@@ -131,10 +142,31 @@ review process.
 Other ways to contribute
 ========================
 
-There are many ways to contribute other than contributing code.  Participating
-in discussions on the scipy-user and scipy-dev *mailing lists* is a contribution
-in itself.  The `scipy.org`_ *website* contains a lot of information on the
-SciPy community and can always use a new pair of hands.
+There are many ways to contribute other than contributing code.
+
+Reviewing open pull requests (PRs) is very welcome, and a valuable way to help
+increase the speed at which the project moves forward.  Everyone who feels like
+they are competent to comment on a pull request is invited to do so; if you
```

Comment:
Thanks @shaunagm! I'll try to rephrase this and expand with a list of things to check for like the OpenDataKit PR.